Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Image/StreamingImageAsset.h

```diff
@@ -127,6 +131,9 @@ namespace AZ
             uint32_t m_totalImageDataSize = 0;
 
             StreamingImageFlags m_flags = StreamingImageFlags::None;
+
+            // Cached value of the average color of this image (alpha-weighted average in case of 4-component images)
+            AZ::Color m_averageColor;
```

Comment:
IIRC the Color constructor does not initialize its data. We should initialize this to black in the constructor.